src/graphql/resolvers/resolvers.ts

```diff
@@ -0,0 +1,24 @@
+import { EmployeeModel } from "../../models/employee";
+
+// use mongoose model to get all employees.
+const getEmployees = async () => EmployeeModel.find({});
+
+// get only one employee by id with mongoose.
+const getEmployee = async (root, data: any) =>
+  EmployeeModel.findOne({ _id: data._id });
+
+const updateEmployee = async (root, { _id, input }) =>
+  EmployeeModel.findOneAndUpdate({ _id }, input);
+
+//load resolvers with the functions related to the schemas of graphql.
+const resolvers = {
+  Query: {
```

Comment:
@EhudPer 
Query -> why with capital letter?

Reply:
This is the convention with graphQL, like in the schema it is with 'Query'